tests/test_ess_csc.py

```diff
@@ -671,3 +671,63 @@ async def test_lightning_data_client_nominal(self) -> None:
                 topic=self.remote.evt_sensorStatus,
                 sensorName="EssElectricField",
             )
+
+    async def test_weather_station_data_client_loses_connection(self) -> None:
+        """The CSC should fault when the RPiDataClient loses its connection
```

Comment:
Please fix the doc string -- wrong data client class.

Also please aim for brief description (one line if possible), blank line, fuller description if needed (in all Python doc strings).

Reply:
I generalized it to `DataClient` and added a brief description. The same doc string now is used for the other unit test methods that test data client connection loss.